Showcase-iOSTests/Dashboard/Presenter/HomePresentableTests.swift

```diff
@@ -33,33 +34,40 @@ class HomePresentableTests: XCTestCase {
         homePresenter.firebaseStorage = mockFirebaseStorage
         homePresenter.signOutInteractor = mockSignOutInteractor
         homePresenter.userDefaults = mockUserDefaults
-        systemUnderTest = homePresenter
+        homePresenter.analyticManager = mockAnalyticsManager
+        presenterUnderTest = homePresenter
+    }
+
+    override func tearDown() {
```

Comment:
Please remove the tear down if you are not using it for anything

Reply:
done :)